### ISSUE TITLE
Submission Title not updating when creating submission from CP

### ISSUE THREAD

**Issue by JeanLucEsser** [verbb/formie#1070]
### Describe the bug

If using a form's field as part of the submission title, it is not reflected in the title when the submission is created from the CP.

Let's say our title format is `timestamp - Name: LastName Name: FirstName`, when we add a submission from the CP, the submission is initiated with `timestamp - ` as its title, and is not updated with the last and first name when completed and saved.

The title field should ideally work the exact same way that the slug field works for Craft entries, updating live until the entry is saved, and should be regenerated upon saving if cleared.

### Steps to reproduce

1. Change the title format for the form's submission to include some form fields (name for example)
2. Create a new submission from the CP
3. The title is not updated with the form fields upon saving

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.2.3

### Plugin version

2.0.8

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.8",
  "...": "..."
}
```

Then run `composer update`. 

A title is always overridable, and doesn't act like a slug. It has its default value set from the Submission Title Format, but you're free to change this as required. This is useful for some people, but means that once created, the title isn't "synced" with the title format.

Changing this behaviour now would be a breaking change.

**Comment by engram-design** [comment]
Fixed in [2.0.9](https://github.com/verbb/formie/releases/tag/2.0.9)
